### ISSUE TITLE
Trying to add multiple items to cart no longer works if ANY of the items included are out of stock

### ISSUE THREAD

**Issue by retrac752** [Ataraksia/NeweggBot#27]
Adding multiple item numbers to the item_number field in the config.json will now add no items to the cart if ANY of the item numbers included are out of stock. You can test this yourself by using the url https://secure.newegg.com/Shopping/AddtoCart.aspx?Submit=ADD&ItemList= 
and then adding an item number for an item in stock, then comma, then an item number for something out of stock. No item will get added. I don't think this is how it worked previously, I think newegg JUST changed this in the last 24 hours.

Could someone else verify they are seeing the same behavior and does anyone have a solution?

**Comment by Codolophier** [comment]
This is correct.  The Newegg endpoint mentioned seems to only add the items in the **ItemList** parameter if they are all available.  Otherwise, none are added.

**Comment by technaustin** [comment]
@Codolophier, is this why when my bot is running, and something comes in stock, it says to verify my info again? I keep getting a pop up about an N95 mask, so I wonder if that is why it cant find the pay button? I have verified, when I make a purchase manually, after checking out, it let's me pay directly, with only having to enter my CVC code, and pressing pay.

**Comment by Codolophier** [comment]
@technaustin Please check that your Newegg [account defaults are all in place](https://github.com/Ataraksia/NeweggBot/issues/26#issuecomment-744044761).  I haven't seen the N95 Mask pop-up, so it is hard to say. Also, which branch you are using?

**Comment by technaustin** [comment]
> @technaustin Please check that your Newegg [account defaults are all in place](https://github.com/Ataraksia/NeweggBot/issues/26#issuecomment-744044761). I haven't seen the N95 Mask pop-up, so it is hard to say. Also, which branch you are using?

Using your master branch. Defaults are set, when I go to buy an item manually it takes me directly to the pay screen. It looks like it was updated a couple days ago, so trying that now.

**Comment by NickFurlo** [comment]
I'm having the same issue on the master branch.

**Comment by ShinmaNoKodou** [comment]
Ah, that explains why my test run with an in-stock memory card + other item numbers out of stock didn't do anything.
